src/masonite/inertia/middleware/InertiaMiddleware.py

```diff
@@ -12,23 +12,23 @@ def __init__(self, request: Request, response: Response):
         self.response = response
 
     def before(self):
-        self.request.is_inertia = self.request.header("HTTP_X_INERTIA")
+        self.request.is_inertia = self.request.header("X_INERTIA")
         if (
             self.request.is_inertia
             and self.request.method == "GET"
-            and self.request.header("HTTP_X_INERTIA_VERSION") != inertia_asset_version()
```

Comment:
Was actually a typo here lol

Reply:
didn't get it ? where was the typo ?
Also shouldn't request be defined ? because self.request is not defined anymore ?

Reply:
nvm not a typo lol. I prob just forgot how to spell inertia after seeing it 100 times 😆